src/chains/filecoin/filecoin/tests/api/filecoin/messages.test.ts

```diff
@@ -1544,5 +1550,98 @@ describe("api", () => {
         assert.ok(result);
       });
     });
+
+    describe("Message Pool Control Functions", () => {
+      let provider2: FilecoinProvider;
+      let client2: LotusClient;
+      let accounts2: Account[];
+      let expectedHeight2 = 0;
+
+      before(async () => {
+        provider2 = await getProvider({
+          chain: {
+            ipfsPort: 5003
+          },
+          miner: {
+            blockTime: 120 // effectively disable mining
```

Comment:
I think a later PR fixes this, but just in case it didn't. Here's a comment.

Reply:
Yup, it's fixed in https://github.com/trufflesuite/ganache-core/pull/842/files#diff-c7da285c1535fab8cd79bef870c259bda7e6a3de7dcfe95b5bd9f41f9e995c10R1566